src/xx.ts

```diff
@@ -272,6 +277,7 @@ export class XXHandshake {
   }
 
   private async readMessageA(hs: HandshakeState, message: MessageBuffer) : Promise<bytes> {
+    console.log("publci key: ", message.ne)
```

Comment:
remove console. You should probably add npm package `debug` and log stuff (separate PR probably)